ghost/core/core/server/data/migrations/versions/5.13/2022-09-02-12-55-rename-members-bio-to-expertise.js

```diff
@@ -0,0 +1,16 @@
+const logging = require('@tryghost/logging');
+const {createNonTransactionalMigration} = require('../../utils');
+
+module.exports = createNonTransactionalMigration(
+    async function up(knex) {
```

Comment:
Can you add log lines in each of the `up` and `down`, just so it's obvious we're running this migration?